worker/worker.go

```diff
@@ -91,6 +91,17 @@ func ReplayWorkflowHistory(logger *zap.Logger, history *shared.History) error {
 	return internal.ReplayWorkflowHistory(logger, history)
 }
 
+// ReplayWorkflowHistoryFromJSONFile executes a single decision task for the given history.
+// Use for testing the backwards compatibility of code changes and troubleshooting workflows in a debugger.
+//
+// The logger is an optional parameter. Defaults to the noop logger.
+// The response contains the decisions produced processing the decision task. It is either
```

Comment:
There is no response besides error in this function signature.

Reply:
fixed the comment